.github/workflows/test.yml

```diff
@@ -40,3 +37,16 @@ jobs:
       - name: Test Format
         run: cargo fmt -- --check
       - run: cargo clippy --features unstable --tests -- -Dclippy::all
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Setup Windows toolchain"
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+      - uses: Swatinem/rust-cache@v1
+      - run: cargo check -p near-sdk --features unstable
+      - run: cargo check -p near-contract-standards
```

Comment:
shouldn't we be adding `--target wasm32-unknown-unknown` to these checks as well?

Reply:
yeah fair point, under the assumption that there can't be a case where this succeeds and wasm32 fails only for windows, but probably safe to add